### ISSUE TITLE
Ability remove labels reported by clients

### ISSUE THREAD

**Issue by hj0x3** [google/grr#492]
Currently, client reported labels remain associated with the client even if a new repack/build is deployed to the client with a different 'baked' in label. It would be beneficial to provide a means to remove or update labels reported from clients, either through some logic or by adopting new labels on client/repack update install and dropping the old.

A use-case for this functional is to allow for tracking custom client builds or repacks via labels, updating revisions as new client builds/repacks are deployed (while under the same GRR version).

**Comment by clairmont32** [comment]
Received the following error message when attempting to replace a label on a client.

`can only concatenate list (not "RepeatedFieldHelper") to list `

**Comment by mbushkov** [comment]
@clairmont32 - quick question: which version of GRR are you using?

**Comment by clairmont32** [comment]
Latest - 3.3.0.0. Just did an install for a POC with a slightly modified config for mail notifications and such; nothing major.

**Comment by clairmont32** [comment]
To be clear, I was doing this through the GUI and not injecting it in the config through `grr_client_build`

**Comment by mbushkov** [comment]
@clairmont32 - thanks for reporting this! Seems like a bug, we'll look into it.

By "replacing label" do you mean deleting a label and then adding a new one, right?

**Comment by clairmont32** [comment]
@mbushkov Correct. I applied an individual label for each of our 2 clients and ended up wanting to change one to test functionality. Went in to see the clients by clicking `search`, selected the one I wanted to make a change to and hit `remove labels`, selected the one I no longer wanted applied and hit `proceed` which returned the above error.
